src/controllers/krate/publish.rs

```diff
@@ -267,3 +268,102 @@ fn parse_new_headers(req: &mut dyn Request) -> CargoResult<(EncodableCrateUpload
     let user = req.user()?;
     Ok((new, user.clone()))
 }
+
+lazy_static! {
+    static ref VERIFIED_EMAIL_REQUIRED_DATE: DateTime<Utc> =
+        { Utc.ymd(2019, 3, 1).and_hms(0, 0, 0) };
```

Comment:
Since this is only used in one place, and the functions being called have basically no overhead, do you think it's worth inlining this to not have to add lazy-static just to remove it next week?

Reply:
Can do!